lib/render/SliceRenderer.cpp

```diff
@@ -173,11 +179,15 @@ int SliceRenderer::_regenerateSlice()
     // Get data values from a slice
     std::unique_ptr<float> dataValues(new float[_textureSideSize * _textureSideSize]);
     planeDescription       pd;
+    pd.sideSize = _textureSideSize;
     pd.origin = {_cacheParams.xOrigin, _cacheParams.yOrigin, _cacheParams.zOrigin};
     pd.rotation = {_cacheParams.xRotation, _cacheParams.yRotation, _cacheParams.zRotation};
     pd.boxMin = _cacheParams.boxMin;
     pd.boxMax = _cacheParams.boxMax;
-    RegularGrid *slice = SliceGridAlongPlane(grid3d, pd, _textureSideSize, dataValues, _windingOrder, _rectangle3D);
+    pd.domainMin = _cacheParams.domainMin;
+    pd.domainMax = _cacheParams.domainMax;
+    RegularGrid *slice = SliceGridAlongPlane(grid3d, pd, dataValues, _windingOrder, _rectangle3D);
+    if (slice == nullptr) return 0;
```

Comment:
Isn't this an error condition? Shouldn't it be returning -1, not 0?

Reply:
Yes. I missed cleaning this up while debugging.  Fixed.